<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences_Widget</class>
 <widget class="QWidget" name="Preferences_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="configSidebar" native="true">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="configStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_5">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Datenbank-Typ:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmb_typ">
         <item>
          <property name="text">
           <string>PostgreSQL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SQLite</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Server:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txt_db_server"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Benutzername:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txt_db_user"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Passwort:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txt_db_pass">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Datenbank:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="txt_db_database"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_mail_server"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Benutzername:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txt_mail_user"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Passwort:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txt_mail_pass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>E-Mail (Absender):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txt_mail_email"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Absender-Name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txt_mail_from"/>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_7">
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Benutzername:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_toa_user"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Passwort:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txt_toa_pass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8">
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Erstellt durch...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txt_by"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Iconmodus:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chk_icon">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Barcodes:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="chk_barcode">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Barcode Schriftart:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFontComboBox" name="cmb_font"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Namenformat Ausdruck:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cmb_nameformat">
         <item>
          <property name="text">
           <string>Vorname Nachname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NACHNAME Vorname</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nachname, Vorname</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Jury-Wertungen runden auf:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cmb_jury">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>0,5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0,05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0,005</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="act_db">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../resources/TurnFix.qrc">
     <normaloff>:/appicons/database.png</normaloff>:/appicons/database.png</iconset>
   </property>
   <property name="text">
    <string>Datenbank</string>
   </property>
  </action>
  <action name="act_mail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../resources/TurnFix.qrc">
     <normaloff>:/appicons/email.png</normaloff>:/appicons/email.png</iconset>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="act_online">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../resources/TurnFix.qrc">
     <normaloff>:/appicons/toa.png</normaloff>:/appicons/toa.png</iconset>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
  </action>
  <action name="act_misc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../resources/TurnFix.qrc">
     <normaloff>:/appicons/misc.png</normaloff>:/appicons/misc.png</iconset>
   </property>
   <property name="text">
    <string>sonstiges</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/TurnFix.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmb_typ</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
